infra/bracin.tf

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+resource "azurerm_linux_virtual_machine" "bracin" {
+  name                  = "bracin"
+  location              = azurerm_resource_group.daml-ci.location
+  resource_group_name   = azurerm_resource_group.daml-ci.name
+  network_interface_ids = [azurerm_network_interface.bracin.id]
+  size                  = "Standard_DS1_v2"
+
+  os_disk {
+    caching              = "ReadOnly"
+    storage_account_type = "Standard_LRS"
+    disk_size_gb         = "30"
```

Comment:
I don't know if you can ask for less but you probably don't need that much?

Reply:
Right; I'll admit I started with a copy/paste of the daily-reset machine and didn't review that section. I don't think we can get away with that much less though as this is a full Ubuntu install. I'll try reducing to 10gb and see if that works.

Reply:
Turns out that number wasn't random: it is the size of the disk in the image we're building from. Azure is happy to give us a disk bigger than the image, but understandably not a smaller one.

Reply:
Wow, unbuntu has gotten bigger in the last 15 years :)

Reply:
It's the size disk image they provide, not the size of the files on that disk. The VM creation system is agnostic to disk format so can't resize down.